Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this for

Reply:
The Info.plist is a necessary file for macos, detailing how applications should be run n stuff.
In case you're curious about a deeper description: https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html

Reply:
lmao I meant why is it here

Reply:
Eto automatically bundles that into the compiled application. It needs to be there, otherwise it won't boot on mac

Reply:
> It needs to be there

So Eto doesn't read it from the project directory?

Reply:
OH! Only just noticed that this is root and not the project directory. Accidental leftover from when I originally created the project in this folder